### DISCUSSION TITLE
Thomas Sandmann’s blog - Figure size, layout & tabsets with Quarto

### DISCUSSION THREAD

**Discussion by giscus** [tomsing1/blog#7] [Announcements]
# Thomas Sandmann’s blog - Figure size, layout & tabsets with Quarto

https://tomsing1.github.io/blog/posts/quarto-figure-size-and-layout/index.html

**Comment by barryrowlingson** [comment]
Is `#| figh-height: 5` a typo? Should it be `#| fig-height: 5` ?

**Reply by tomsing1** [reply]
Yes, indeed! Thank you for pointing it out. I will fix it!
